config/pycryptobot.yml.example

```diff
@@ -0,0 +1,46 @@
+### PyCryptoBot Configuration Settings
+---
+
+pycryptobot:
+
+### Toggle application level debug (does not toggle framework debugging)
+# debug: false
+
+### Where external (third-party) plugins are loaded from
+# plugin_dir: /var/lib/pycryptobot/plugins/
+
+### Where all plugin configurations are loaded from
+# plugin_config_dir: /etc/pycryptobot/plugins.d/
+
+### Where external templates are loaded from
+# template_dir: /var/lib/pycryptobot/templates/
+
+### The log handler label
+# log_handler: colorlog
+
+### The output handler label
+# output_handler: jinja2
+
+### sample foo option
+# foo: bar
+
+
+log.colorlog:
```

Comment:
a mapping key should not have a dot in its name. dots are usually used as separators, so it would have to be escaped and it would just become ugly. can we do
```
log:
  colorlog:
    colorlog_option1: true
  consolelog:
    consolelog_option1: false
  telegram:
    telegram_option1: true
```

Reply:
Is autogenerated by cement, and looks like it should be with "." in the name. I'm not sure

Reply:
hmm ok, weird.

Reply:
[Here](https://docs.builtoncement.com/core-foundation/configuration-settings#configuration-file-loading) how  they're working with configs